docs/workloads/azure/data/etl_pipelines/datastacks.md

```diff
@@ -21,14 +21,13 @@ Datastacks is a utility built to support various functions within the Ensono Sta
 * [Setup project environment](../getting_started/dev_quickstart_data_azure.md)
 
 ```bash
-# Initiate Datastacks using poetry:
-poetry run datastacks
-
-# Run Datastacks from the command line
+# Option 1: Run Datastacks CLI using Poetry's interactive shell (preferred)
+poetry shell
 datastacks --help
+exit
```

Comment:
Any reason we need to have this exit here? Presume people would typically keep the shell open while developing

Reply:
It was to demonstrate where shell isn't required for the second example but the poetry run version will still work if we keep it out (as I have done now)